Assets/Scripts/ProgressManager.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[System.Serializable]

```

Comment:
Why do we need to make this serializable?

Reply:
Oops forgot to take that out